experiments/memcached-cat/main.go

```diff
@@ -100,8 +99,16 @@ func main() {
 	}
 
 	// Discover local CPU topology.
+	hpCores := topology.HpRangeFlag.Value()
 	beCores := topology.BeRangeFlag.Value()
+
+	if len(beCores) == 0 || len(hpCores) == 0 {
```

Comment:
You could have used flag.Name() so that you will be safe when flag name changes.